source/js/main.js

```diff
@@ -241,8 +243,8 @@ let main = {
     }
 
     // Embed additional instances of the Join Us box that don't need an API exposed (eg: Homepage)
-    if (document.querySelectorAll(`.join-us`)) {
-      var elements = Array.from(document.querySelectorAll(`.join-us`));
+    if (document.querySelectorAll(`.join-us:not(.on-nav)`)) {
```

Comment:
If we're touching this, let's make sure we don't run the same query selector twice:
```
var joinUsElements = document.querySelectorAll(`.join-us:not(.on-nav)`);
if (joinUsElements.length) {
  ..
}
```